services/src/test-library/src/main/java/org/openkilda/testing/model/topology/TopologyDefinition.java

```diff
@@ -57,6 +61,7 @@
     private List<TraffGen> traffGens;
     @NonNull
     private TraffGenConfig traffGenConfig;
+    private final String controller = VIRTUAL_CONTROLLER_ADDRESS;
```

Comment:
This value should probably be handled via config file, because "tcp:kilda:6653" won't resolve on the LabService's side